### ISSUE TITLE
GitHub Actions tests failing on Ubuntu due to issue with conda setup

### ISSUE THREAD

**Issue by ethanwhite** [weecology/DeepForest#323]
See https://github.com/weecology/DeepForest/runs/5753610560?check_suite_focus=true#step:4:314

@henrykironde - can you take a look at this?

**Comment by ethanwhite** [comment]
Actually @henrykironde - let's wait on this until #314 is in. I forgot that I probably fixed it as part of getting the tests working on that PR, so hopefully once it's merged the CI will go back to working.

**Comment by ethanwhite** [comment]
@bw4sz - I think this makes merging #314 (or having me split it into two pieces and merging the testing piece) a priority

**Comment by bw4sz** [comment]
Merged. Close when ready.
